### DISCUSSION TITLE
Add followed only page just like the Youtube subscription page

### DISCUSSION THREAD

**Discussion by wavjav** [civitai/civitai#463] [Ideas]
Considering a lot of users here hating about stuff they don't like on Civitai shows up on their front page and it messes up their searching/discovering here..
How about making a new page/option just like the Subscription page on Youtube
On this page, you can only see any models that have been uploaded by creators that you follow only

With this option, now you can see any models/style that you like from creators you follow, no bs about finding your beloved creators' newest uploads in the sea of p#rn and weird fetish uploaded by random new users who still learn about training their model or sh#t

**Comment by mitchoz** [comment]
Would love to see this as well, but why would you limit it to Models?
Maybe I'm mistaken but I think there is no general way to display any kind of content from creators you follow. So it would make sense to just add it as a filter option in the top navigation?

![image](https://github.com/civitai/civitai/assets/3161448/51e514fb-0903-4d8f-bf88-4ae4eca15328)
